presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/adapters/RecyclerViewAdapter.java

```diff
@@ -0,0 +1,68 @@
+package com.globant.equattrocchio.cleanarchitecture.mvp.view.adapters;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.globant.equattrocchio.cleanarchitecture.R;
+import com.globant.equattrocchio.domain.enities.ImageEntity;
+
+import java.util.List;
+
+public class RecyclerViewAdapter extends RecyclerView.Adapter<RecyclerViewAdapter.RecyclerViewViewHolder> {
+
+    private Context context;
+    private List<ImageEntity> images;
```

Comment:
Use onlyEntity by BD, and not pass to view

Reply:
done